internal/ice/util.go

```diff
@@ -12,28 +12,39 @@ func localInterfaces() (ips []net.IP) {
 	}
 
 	for _, iface := range ifaces {
-		if iface.Flags&net.FlagUp == 0 {
```

Comment:
The old way is easier to parse IMO. By putting them in the same conditional I took a second parsing them trying to figure out if they were related.

I would keep as two if branches or `if/elsif`

Reply:
Changing it. I was just copy pasting from my version of the library.